astronomer/providers/amazon/aws/triggers/redshift_cluster.py

```diff
@@ -46,26 +45,28 @@ async def run(self):
         if operation_type is 'pause_cluster it will call the pause_cluster function in RedshiftHookAsync
         """
         hook = RedshiftHookAsync(aws_conn_id=self.aws_conn_id)
-        while True:
-            try:
-                if self.operation_type == "resume_cluster":
-                    response = await hook.resume_cluster(cluster_identifier=self.cluster_identifier)
-                    if response:
-                        yield TriggerEvent(response)
-                        return
-                    else:
-                        error_message = f"{self.task_id} failed"
-                        raise AirflowException(error_message)
```

Comment:
I thought only tests are supposed to be fixed.Why are code changes being done to address code coverage. Did we miss any code during the merge ?